### PR TITLE
Second comparison script

### PR DESCRIPTION
**Pull request by NNoureen**
### Purpose/implementation Section
The analysis is focused on identifying telomerase activities using gene expression data sets processed in different ways, to identify platform differences and compare the telomerase activities across different tumor histologies.

#### What was your approach?
We used our newly developed method EXTEND (Expression based Telomerase ENzymatic activity Detection).

#### What GitHub issue does your pull request address?
Issue # 148 (Quantify telomerase activity across pediatric brain tumors)

#### Which areas should receive a particularly close look?
We would like the reviewers to provide us directions for further comparisons regarding telomerase activities if required.
#### Is the analysis in a mature enough form that the resulting figure(s) and/or table(s) are ready for review?
YES
#### What types of results are included (e.g., table, figure)?
Predicted Telomerase activity scores as tables and comparisons as figures.
#### What is your summary of the results?
Results include text files containing telomerase activities across different expression data formats. The comparison of telomerase activities has been carried out with different parameters as:
1.	Variations in telomerase activities across different expression data formats.
2.	Comparison of telomerase activities with TERT and TERC genes expression for different data sets.
3.	Distribution of telomerase activities across different pediatric brain tumor histologies.

#### Documentation Checklist

This analysis module has a `README` and it is up to date.
This analysis is recorded in the table in `analyses/README.md` and the entry is up to date.
The analytical code is documented.

### CODE REVIEW

File: analyses/telomerase-activity-prediction/03-Comparing-TERTexp-TERCexp-EXTENDScores.R [hunk 1]
```diff
@@ -0,0 +1,112 @@
+library('ggpubr')
```
Comment (sjspielman) [inline analyses/telomerase-activity-prediction/03-Comparing-TERTexp-TERCexp-EXTENDScores.R]: Most of these are actually not needed. You can replace lines 1-6 with _only_:
```
library('ggpubr')
stringsAsFactors=FALSE
library(grid)
```

**Comment by sjspielman** [comment]
Looks totally good to me! I'd recommend paring down the loaded libraries like last time, but otherwise this seems all set!

**Comment by NNoureen** [comment]
Sure. Thanks

**Comment by NNoureen** [comment]
Sure. I will just do it and commit the change

**Comment by NNoureen** [comment]
Cleaned up the libraries

**Comment by NNoureen** [comment]
Thankyou all.

**Pull request merged**